notflix

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
 query=$(printf '%s' "$*" | tr ' ' '+' )
-movie=$(curl -s https://1337x.to/search/$query/1/ | grep -Eo "torrent\/[0-9]{7}\/[a-zA-Z0-9?%-]*/" | head -n 1)
+movie=$(curl -s https://1337x.to/search/$query/1/ | grep -Eo "torrent\/[0-9]*\/[a-zA-Z0-9?%-]*/" | head -n 1)
 magnet=$(curl -s https://1337x.to/$movie | grep -Po "magnet:\?xt=urn:btih:[a-zA-Z0-9]*" | head -n 1)
-peerflix -l -k $magnet
+peerflix -l -k $magnet --vlc
```

Comment:
why is vlc hard coded?

Reply:
Also, it's wrongly coded, it will open up both mpv and vlc. Remove -k if you only want vlc.